src/sentry/api/permissions.py

```diff
@@ -28,27 +29,41 @@
     from sentry.models.organization import Organization
 
 
-class RelayPermission(permissions.BasePermission):
+class RelayPermission(BasePermission):
     def has_permission(self, request: Request, view: object) -> bool:
         return getattr(request, "relay", None) is not None
 
 
-class SystemPermission(permissions.BasePermission):
+class SystemPermission(BasePermission):
     def has_permission(self, request: Request, view: object) -> bool:
         return is_system_auth(request.auth)
 
 
-class NoPermission(permissions.BasePermission):
+class NoPermission(BasePermission):
     def has_permission(self, request: Request, view: object) -> bool:
         return False
 
 
-class SuperuserPermission(permissions.BasePermission):
+class SuperuserPermission(BasePermission):
     def has_permission(self, request: Request, view: object) -> bool:
         return is_active_superuser(request)
 
 
-class ScopedPermission(permissions.BasePermission):
+class StaffPermission(BasePermission):
+    def has_permission(self, request: Request, view: object) -> bool:
+        return is_active_staff(request)
+
+
+class SuperuserOrStaffFeatureFlaggedPermission(BasePermission):
```

Comment:
should we add a note for what this is in case other folks come across it and get confused?